src/openvic-dataloader/csv/Parser.cpp

```diff
@@ -166,7 +176,7 @@ bool Parser::parse_csv(bool handle_strings) {
 	}();
 	_has_error = _parse_handler->parse_state().logger().errored();
 	_has_warning = _parse_handler->parse_state().logger().warned();
-	if (!errors->empty()) {
+	if (errors->empty()) {
```

Comment:
This change doesn't make sense to me.
If errors are empty, _has_error should be false.
So the original was correct.